core/domain/fs_domain.py

```diff
@@ -108,96 +107,105 @@ def version(self):
         return self._version
 
 
-class ExplorationFileSystem(object):
-    """A datastore-backed read-write file system for a single exploration.
+class EntityFileSystem(object):
+    """A datastore-backed read-write file system for a single entity.
 
-    The conceptual intention is for each exploration to have its own asset
-    folder. An asset has no meaning outside its exploration, so the assets in
-    these asset folders should therefore not be edited directly. They should
-    only be modified as side-effects of some other operation (such as adding an
-    image to an exploration).
-
-    The content of an exploration should include a reference to the asset
-    together with the version number of the asset. This allows the
-    exploration to refer to asset versions.
+    The conceptual intention is for each entity type to have its own parent
+    folder. In this, each individual object will have its own folder with the
+    corresponding ID as the folder name. These folders will then have the assets
+    folder inside which stores images, audio etc (example path:
+    story/story_id/assets/).An asset has no meaning outside its entity, so the
+    assets in these asset folders should therefore not be edited directly. They
+    should only be modified as side-effects of some other operation (such as
```

Comment:
Perhaps: "... some other operation on their corresponding entity (such as adding an image to that entity)."

Reply:
Done